### ISSUE TITLE
🐞 Bug Report: Elementor Pro 3.12 Popup Background Color Bug - [ED-10300]

### ISSUE THREAD

**Issue by mcrowell** [elementor/elementor#21824]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Elementor 3.12 is causing issues for the background colors of Popups. 

I created a detailed video on the bug here:
https://www.youtube.com/watch?v=5GTahjYxRMM

### Steps to reproduce

1. Create a Popup and don't assign a background color to the popup or section.
2. View the Popup in a new Incognito window or in a preview.
3. You will see the Popup background is not rendering correctly.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MySQL Community Server (GPL) v5.7.41
	PHP Version: 8.1.17
	PHP Memory Limit: 256M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://wickydemo.com/testing
	Home URL: https://wickydemo.com/testing
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 128M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/New_York
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Code Snippets Pro
		Version: 3.3.0
		Author: Code Snippets Pro

	Elementor
		Version: 3.12.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.0
		Author: Elementor.com

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence


== Elements Usage ==
	
	header : 2
		html : 1
		nav-menu : 1
	loop : 0
		text-editor : 1
	page : 28
		button : 38
		divider : 30
		form : 7
		heading : 160
		html : 16
		icon : 20
		icon-list : 4
		image : 54
		template : 3
		text-editor : 58
		toggle : 3
		video : 1
	popup : 13
		button : 1
		form : 1
		heading : 3
		icon : 1
		nav-menu : 1
		social-icons : 1
		spacer : 2
		text-editor : 2
		theme-site-logo : 1
	section : 22
		image : 2
		nav-menu : 1
		price-list : 3
		text-editor : 2
		video : 3
	wp-post : 8
		text-editor : 1
	wp-page : 236
		button : 53
		call-to-action : 1
		code-highlight : 2
		divider : 8
		form : 11
		gallery : 2
		heading : 151
		html : 36
		icon : 10
		icon-box : 7
		icon-list : 12
		image : 113
		lottie : 1
		nav-menu : 2
		paypal-button : 1
		posts : 2
		search-form : 1
		shortcode : 4
		social-icons : 1
		spacer : 36
		stripe-button : 1
		template : 9
		text-editor : 117
		theme-site-logo : 1
		toggle : 5
		video : 8
	single-post : 1
		button : 1
		heading : 1
		post-navigation : 1
		share-buttons : 1
		spacer : 1
		template : 1
		text-editor : 2
		theme-post-content : 1
		theme-post-title : 1


== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 1
	Custom Icons: 3

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimized DOM Output: Inactive
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive
	Accessibility Improvements: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Inactive
	Lazy Load Background Images: Inactive
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Inactive
	Scroll Snap: Inactive
	Menu: Inactive by default




````

</details>

**Comment by MisterPascal** [comment]
I have the same problem! Since 3.12 no popup is displayed correctly....

**Comment by Michaelgimii** [comment]
As a small popup issue, in the pop-up view conditions section, the **Show up to X times** switcher button is out of center.

![1](https://user-images.githubusercontent.com/88034902/228938793-d25cf6e7-d9e5-408c-a712-a8e28818a4f0.png)

**Comment by trixxmanaty** [comment]
I am also experiencing the same issue. The pop-up menu is not displaying well after updating to the version 3.12

**Comment by nicholaszein** [comment]
Hello everyone! 

Thank you for submitting your issue! 🙏 

We are aware that the new color scheme of Elementor 3.12 is affecting some styles that it shouldn't be affecting. It seems it only happens when the user is logged in.

A fix is underway.

### 🔗 🐞 This issue is Related to #21809 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

Best regards

**Comment by nicholaszein** [comment]
Apologies,

previously I had marked this issue as a duplicate of issue #21809. This was incorrect.

I've reopened this issue, and a task to fix it was already created. We are investigating and will update you as soon as we have more information.

Thank you for your feedback, and sorry for the confusion! 🙏

**Comment by arielk** [comment]
This issue has been resolved in Elementor Pro v3.12.1

Feel free to update

Thanks everyone!
